libmachine/mcnutils/b2d.go

```diff
@@ -20,12 +19,8 @@ var (
 	GithubApiToken string
 )
 
-const (
-	timeout = time.Second * 5
-)
-
 func defaultTimeout(network, addr string) (net.Conn, error) {
-	return net.DialTimeout(network, addr, timeout)
+	return net.Dial(network, addr)
```

Comment:
We can't guess the duration of the download. I'm currently on a slow network and it takes more than 5 minutes to download.